### PR TITLE
14 test examples 13 15

### PR DESCRIPTION
**Pull request by lojzatran**
Additionally, I also edited the Example 5, because it was not according to our spec anymore.

Fixes: https://github.com/commercetools/commercetools-node-variant-reassignment/issues/14

### CODE REVIEW

File: spec/integration/runner/variant-reassignment-example-13.spec.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    }], [product1])
+    const { body: product } = await ctpClient.products.byId(product1.id).fetch()
+    expect(product.masterData.current.masterVariant.sku).to.equal('1')
+    expect(product.masterData.current.variants.length).to.equal(0)
+    expect(product.masterData.staged.masterVariant.sku).to.equal('1')
+    expect(product.masterData.staged.variants.length).to.equal(0)
+    const { body: { results } } = await utils.getProductsBySkus(['2', '3', '4'], ctpClient)
+    expect(results).to.have.lengthOf(1)
+    const updatedProductProjection = results[0]
+    expect(updatedProductProjection.masterVariant.sku).to.equal('2')
```
Comment (junajan) [inline spec/integration/runner/variant-reassignment-example-13.spec.js]: should we also check if the product is anonymized with `_ctsd` slug?

Comment (junajan) [inline spec/integration/runner/variant-reassignment-example-13.spec.js]: should we also check if the product is anonymized with `_ctsd` slug?

File: spec/integration/runner/variant-reassignment-example-14.spec.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+  it('different products on staged and current AND different variants on staged and current',
+    async () => {
+      const reassignment = new VariantReassignment([], logger, {})
+      await reassignment.execute([{
+        productType: product1.productType,
+        name: product1.name,
+        slug: product1.slug,
+        masterVariant: product1.masterVariant,
+        variants: [{}]
```
Comment (junajan) [inline spec/integration/runner/variant-reassignment-example-14.spec.js]: Here the productDraft does not have any variants but in the documentation above and in the `except()` bellow we are expecting variant v2.

File: spec/integration/runner/variant-reassignment-example-15.spec.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+import * as utils from '../../utils/helper'
```
Comment (junajan) [inline spec/integration/runner/variant-reassignment-example-15.spec.js]: It looks like this example 15 is pretty much the same as in example 13 - again we are dealing with different staged and current variants so I was wondering if this test is needed?

File: spec/integration/runner/variant-reassignment-example-15.spec.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+    let product1Response = await utils.createCtpProducts([['5']], ctpClient)
+    product1 = product1Response[0]
+    product1Response = await ctpClient.products
+      .byId(product1.id)
+      .update({
+        version: product1.version,
+        actions: [{ action: 'publish' }]
+      })
+    product1Response = await ctpClient.products
```
Comment (junajan) [inline spec/integration/runner/variant-reassignment-example-15.spec.js]: You can merge these three actions below into one request.

File: spec/integration/runner/variant-reassignment-example-15.spec.js (hunk 3 above)
Comment (lojzatran) [reply spec/integration/runner/variant-reassignment-example-15.spec.js]: Yes, it is indeed quite similar. Maybe it's not necessary, but we will decide this in the future?
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/19/commits/dca62eb6e3f90b0d0d9348a361cb29ae7ab52f1a

File: spec/integration/runner/variant-reassignment-example-14.spec.js (hunk 2 above)
Comment (lojzatran) [reply spec/integration/runner/variant-reassignment-example-14.spec.js]: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/19/commits/59c189a49f6261b2e50ca15c8854da4beda4719c

File: spec/integration/runner/variant-reassignment-example-15.spec.js (hunk 4 above)
Comment (lojzatran) [reply spec/integration/runner/variant-reassignment-example-15.spec.js]: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/19/commits/e27316f3b6af91331943c71ba6ca28c1b582ea56

File: spec/integration/runner/variant-reassignment-example-13.spec.js (hunk 1 above)
Comment (lojzatran) [reply spec/integration/runner/variant-reassignment-example-13.spec.js]: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/19/commits/5b95f99660119eacee33f45fed265f97ebcd9aa9

File: spec/integration/runner/variant-reassignment-example-15.spec.js (hunk 3 above)
Comment (junajan) [reply spec/integration/runner/variant-reassignment-example-15.spec.js]: yup yup..

**Comment by junajan** [comment]
@lojzatran do you have any idea why there are broken tests? In master, we already have some `spec/integration/runner/...` tests and the tests are green.

**Comment by lojzatran** [comment]
@junajan the problem is the mocha cmd is wrong and it does not recursively search for all folders. We have:
```js
mocha --recursive spec/**/*.spec.js
```
but this search only first level folder, and because the failing tests are in `/spec/integration/runner`, it does not run them. The correct cmd should be:
```js
mocha --recursive ./spec
```

**Comment by junajan** [comment]
Thx for investigating @lojzatran

**Pull request merged**